apps/Gruntfile.js

```diff
@@ -77,7 +77,7 @@ testsContext.keys().forEach(testsContext);
   var appsToBuild = SINGLE_APP ? [SINGLE_APP] : ALL_APPS;
 
   var ace_suffix = envConstants.DEV ? '' : '-min';
-  var dotMinIfNotDev = envConstants.DEV ? '' : '.min';
+  // var dotMinIfNotDev = envConstants.DEV ? '' : '.min'; <---- linter threw an error that this variable is defined, but never used. Should I delete it?
```

Comment:
I saw the same and am about to delete it when I merge https://github.com/code-dot-org/code-dot-org/pull/14229